minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -280,6 +282,10 @@ public void stop() throws Exception {
 
     resetContext();
     resetLoginUser();
+    if (mHome != null) {
```

Comment:
Is it possible that the cluster crashes before stop? In that case, will we have leftovers in home?